config.py

```diff
@@ -14,9 +14,27 @@ class ConfigEmail:
 
 class Config:
     def __init__(self):
-        config_path = os.getenv("IOT_IDS_CONFIGPATH", "src/config.json")
+        config_path = os.getenv("IOT_IDS_CONFIGPATH", "config.json")
         with open(config_path) as f:
```

Comment:
Why can't this just be `with open("config.json") as f:`?

Reply:
This replaces the path with an environment variable that is defined in cids-start. Which will allow the user to change the config location for their environment without having to touch the config object code.